packages/contentstack-bulk-publish/package.json

```diff
@@ -8,17 +8,15 @@
     "@contentstack/cli-command": "^1.0.0",
     "@contentstack/cli-utilities": "^1.0.0",
     "@contentstack/management": "^1.3.0",
-    "@oclif/command": "^1.8.16",
-    "@oclif/config": "^1.18.3",
+    "@oclif/core": "^1.13.10",
     "bluebird": "^3.7.2",
+    "inquirer": "^8.0.0",
     "lodash": "^4.17.15",
     "nock": "^13.0.2",
-    "winston": "^3.7.2",
-    "inquirer": "^8.0.0"
+    "winston": "^3.7.2"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "^1.22.2",
-    "@oclif/plugin-help": "^5.1.12",
+    "oclif": "^3",
```

Comment:
Can we use the latest oclif verion [3.1.2] ?

Reply:
done